### PR TITLE
feat: Avatar library component

### PR DESCRIPTION
**Pull request by eldhoabraham94**
Adding changes for Avatar library component
Adding prism code higlighter

### CODE REVIEW

File: components/common.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.item-conainter{
```
Comment (nisargpawade) [inline components/common.css:1]: container spelling, do give some padding to main-section class

File: doc/doc.html [hunk 2]
```diff
@@ -33,5 +34,49 @@
             </ul>
         </nav>
     </div>
+
+    <div class="main-section">
+        <h1 class="item-head">Avatar</h1>
+        <p class="item-desc">Show display picture of a user</p>
+        <p class="item-details">To use this avatar, you can use class name 'avatar'. For multiple sizes following classes can be used av-xxl, av-xl, av-lg, av-sm </p>
+        <div class="item-conainter">
```
Comment (nisargpawade) [inline doc/doc.html:42]: container spelling

**Review by nisargpawade — APPROVED** [review]

File: components/common.css (hunk 1 above)
Comment (eldhoabraham94) [reply components/common.css:1]: @nisargpawade Container spelling corrected
Padding added for main section

File: doc/doc.html (hunk 2 above)
Comment (eldhoabraham94) [reply doc/doc.html:42]: @nisargpawade Container spelling corrected

**Pull request merged**